mobify/platform-scaffold
# [Integration Manager] Convert to SDK `createAction` utilities

This PR converts this branch to use createAction from the SDK rather than including it in the scaffold. 

 **JIRA**: Preparation for moving stuff to the SDK
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/561

## Changes
- Remove the `createAction` and `createActionWithMeta` utilities from the scaffold
- Convert all the uses to import from the SDK
- Change to the new API

## How to test-drive this PR
- Link in the associated SDK branch.
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Make sure things work as before.
